### ISSUE TITLE
Recent BN json change - manually adding xl variants is now unnecessary

### ISSUE THREAD

**Issue by ChucklesTheBeard** [NobleJake/ProjectPackRat#6]
```
DEBUG    : (json-error)
Json error: C:/[snip]/mods/ProjectSquirrel/items/sq_armor_override.json:427:3: JSON contains circular dependency, this object is discarded

    "valid_mods": [ "steel_padded", "alloy_padded", "lightened", "pockets" ]
  },
  {
  ^

    "id": "xlsurvivor_suit",
    "type": "ARMOR",


 FUNCTION : void DynamicDataLoader::load_deferred(DynamicDataLoader::deferred_json&)
 FILE     : src/init.cpp
 LINE     : 193
 VERSION  : BN 9ebf08f
```

**Comment by Ignaramico** [comment]
in case someone is wondering, the issue still persist, but you only need to remove the file from the mod

**Comment by NobleJake** [comment]
I'm sorry. I am currently serving in the military in South Korea and don't have access to a computer, so I have limited access to GitHub on my phone. You can download it from the new release at the link below. My collaborators and I fixed all known errors :D

https://github.com/NobleJake/ProjectPackRat/releases/tag/202403310850

**Comment by scarf005** [comment]
should be fixed by #8.
